src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -588,8 +588,8 @@ export default class QuestionnaireForm extends Component {
         console.log(priorAuthBundle);
 
         const Http = new XMLHttpRequest();
-        const priorAuthUrl = "https://davinci-prior-auth.logicahealth.org/fhir/Claim/$submit";
-        // const priorAuthUrl = "http://localhost:9000/fhir/Claim/$submit";
+        // const priorAuthUrl = "https://davinci-prior-auth.logicahealth.org/fhir/Claim/$submit";
+        const priorAuthUrl = "http://localhost:9000/fhir/Claim/$submit";
```

Comment:
Did you mean to check in the change to make localhost the default?

Reply:
This was just for testing since there were changes to PAS. I will update before it is merged